src/Utils/DateUtils.php

```diff
@@ -211,4 +211,114 @@ public static function getLastDayOfMonthYear(int $month, int $year): \DateTime
     {
         return self::getFirstDayOfMonthYear($month, $year)->modify('last day of this month')->setTime(23, 59, 59);
     }
+
+    public static function getFirstDayYearFromDatetime(?\DateTime $datetime = null): \DateTime
```

Comment:
Datetime should be writting a capital T like the object (do this for every function that contains DateTime in it)